canonical/snapcraft.io
# chore: Restructure store to match standardised architecture

## Done
Restructured the app to remove legacy store JS and standardise the app architecture

## How to QA
- Go to https://snapcraft-io-4646.demos.haus/store
- Check that it looks and works as expected ([compare to live](https://snapcraft.io/store))

## Testing
- [x] No testing required (explain why): This is a restructure

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-11334
